conf/modules/prepare_genome.config

```diff
@@ -76,7 +76,7 @@ process {
     }
 
     withName: 'TABIX_DBSNP' {
-        ext.when         = { !params.dbsnp_tbi && params.dbsnp && (params.step == "mapping" || params.step == "markduplicates" || params.step == "prepare_recalibration") || params.tools && (params.tools.split(',').contains('controlfreec') || params.tools.split(',').contains('haplotypecaller') || params.tools.split(',').contains('mutect2')) }
+        ext.when         = { !params.dbsnp_tbi && params.dbsnp && ((params.step == "mapping" || params.step == "markduplicates" || params.step == "prepare_recalibration") || params.tools && (params.tools.split(',').contains('controlfreec') || params.tools.split(',').contains('haplotypecaller') || params.tools.split(',').contains('mutect2'))) }
```

Comment:
Might be missing something but why you need this additional parenthesis? does not seem necessary to me

Reply:
Actually there are 2 condition in the parenthesis, and I only need one of them to be true.
If I don't do so, then it's either the first conditions `(!params.dbsnp_tbi && params.dbsnp)` and the third, or only the fourth

Reply:
which is what was causing the mess in the first place

Reply:
Ah sure now I see it thanks for the explanation! :smile: